.github/workflows/esti.yaml

```diff
@@ -653,6 +654,64 @@ jobs:
         working-directory: test/spark
         run: docker-compose logs --tail=1000 lakefs
 
+  metadata-client-gc-spark3-on-azure:
+    name: Test lakeFS metadata client GC with Spark 3.x on Azure
+    needs: [deploy-image, build-spark3-metadata-client]
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        spark:
+          - version: 3.2.1
+            suffix: "-hadoop3"
+    env:
+      SPARK_TAG: ${{ matrix.spark.version }}
+    steps:
+      - name: Check-out code
+        uses: actions/checkout@v2
+
+      - name: Import Metaclient
+        id: import_metaclient
+        uses: actions/download-artifact@v3
+        with:
+          name: metaclient-3x
+          path: test/spark/metaclient
+
+      - name: Generate uniquifying value
+        id: unique
+        run: echo "::set-output name=value::$RANDOM"
+
+      - name: Start lakeFS for Spark tests
+        uses: ./.github/actions/bootstrap-test-lakefs
+        with:
+          compose-directory: test/spark
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Do we need to keep those here?

Reply:
Yes because we need to login to Amazon ECR.